pdns/dnsdistdist/dnsdist-lua-inspection-ffi.h

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
Maybe an include guard?

Reply:
The reason I have not been doing that is that LuaJIT documentation states:
> C declarations are not passed through a C pre-processor, yet. No pre-processor tokens are allowed, except for #pragma pack. Replace #define in existing C header files with enum, static const or typedef and/or pass the files through an external C pre-processor (once). Be careful not to include unneeded or redundant declarations from unrelated header files. 

So I'm not sure it would work, and if it does that it would work with all versions of LuaJIT.